main.py

```diff
@@ -230,12 +230,21 @@ def parse_course_folder_name(course: canvasapi.canvas.Course) -> str:
         r = r.groupdict()
     else:
         r = {}
+        
+    if 'original_name' in course.__dict__:
```

Comment:
remove `__dict__`

Reply:
https://stackoverflow.com/questions/610883/how-to-know-if-an-object-has-an-attribute-in-python